### PR TITLE
 Remove librdp.so sources, it's obsoleted by neutrinordp

### PR DESCRIPTION
**Pull request by proski**
Rename "neutrinordp-and" section in xrdp.ini to "rdp-any", as it's the only available RDP client now.

The `rdp` directory had no code changes since year 2010 (apart from warning fixes, cleanup and changes affecting the whole source tree).

This fixes issue #592

### CODE REVIEW

**Comment by speidy** [comment]
+1

File: xrdp/xrdp.ini [hunk 1]
```diff
@@ -208,12 +208,6 @@ password=ask
 
 [rdp-any]
 name=rdp-any
-lib=librdp.so
-ip=ask
-port=ask3389
-
-[neutrinordp-any]
-name=neutrinordp-any
```
Comment (speidy) [inline xrdp/xrdp.ini]: that change can make some people confused.
it should be documented.

Comment (proski) [inline xrdp/xrdp.ini]: Actually, I think all "*-any" sections should not be in the default install. They allow using xrdp as a proxy to another network, which it not an expected behavior. Instead, those sections should be in a sample file with extensive documentation.

I guess I'll just keep using `neutrinordp-any` for now, as this PR is about a different issue.

**Comment by proski** [comment]
The change for the section name has been removed from this PR.

**Comment by jsorg71** [comment]
merged as 2b614ed4ab7b2c022325edb1dd32c3d9ca0d088b

**Pull request closed without merging**